examples/AspNetCore/Startup.cs

```diff
@@ -109,6 +128,18 @@ public void ConfigureServices(IServiceCollection services)
                         };
                     });
 
+                    // Configure builder, but also need to get hold of IServiceProvider.
```

Comment:
@alanwest 
From this comment: https://github.com/open-telemetry/opentelemetry-dotnet/pull/1469/files#r518372009